bpf/lib/generic.h

```diff
@@ -68,7 +68,7 @@ struct names_map_key {
 
 struct {
 	__uint(type, BPF_MAP_TYPE_HASH);
-	__uint(max_entries, 64);
+	__uint(max_entries, 256);
```

Comment:
same as before const make it easy to track

Reply:
As this constant is only used here, I believe that it is fine as it is. I added a comment there to describe what this constant means.

Reply:
Oh you are right sorry, thank you!